solution/ui/prototype/src/components/header/SearchInput.vue

```diff
@@ -29,11 +25,21 @@ export default {
                 [this.type]: true,
             };
         },
-    }
-}
+    },
+};
 </script>
 
 <style>
-
+.search-box input {
+    border: none;
+    flex: auto;
+    padding: 30px;
+    background-image: url(https://cdn2.hubspot.net/hubfs/4004166/bioticresearch_website_assets/images/search_icon.png);
```

Comment:
we're using some inline SVGs for this in the header/ToggleButton.vue component - they look much sharper than this PNG.  I also don't think it should be a background image here because text renders on top of it if the search query is long enough:

![Screen Shot 2022-02-16 at 15 33 18](https://user-images.githubusercontent.com/1651907/154352421-b07588fa-930a-419f-a37d-33f11027843d.png)

The styles also don't render well in Firefox.